utilities/transactions/write_unprepared_txn.cc

```diff
@@ -69,14 +71,20 @@ WriteUnpreparedTxn::~WriteUnpreparedTxn() {
           log_number_);
     }
   }
+
+  // Call TransactionBaseImpl::Clear so that ~PessimisticTransaction does not
```

Comment:
This seems complicated to me and prone to errors in future. Can we simply call tracked_keys_.clear()?